core/common/src/ConcurrentQuery.ts

```diff
@@ -239,7 +239,7 @@ export class QueryBinder {
   public bindBlob(indexOrName: string | number, val: Uint8Array) {
     this.verify(indexOrName);
     const name = String(indexOrName);
-    const base64 = Buffer.from(val).toString("base64");
+    const base64 = PlatformUtilities.utilities.toBase64(val);
```

Comment:
why is this in common anyway? We don't want to use base64 on the backend, right?

Reply:
It (and it's unwanted base64 behavior) don't seem to be used on the backend (other than the ecsql tests).

Reply:
good, because when we bind a blob on the backend it needs a Buffer, not a string.